l10nzh.properties

```diff
@@ -2,16 +2,22 @@ category=Localization
 description=SonarQube Chinese Pack
 homepageUrl=https://github.com/SonarCommunity/sonar-l10n-zh
 archivedVersions=
-publicVersions=9.9,10.0,10.1,10.2,10.3,10.4
+publicVersions=9.9,10.0,10.1,10.2,10.3,10.4.1,10.5
 
 defaults.mavenGroupId=org.codehaus.sonar-plugins.l10n
 defaults.mavenArtifactId=sonar-l10n-zh-plugin
 
-10.4.description=Support SonarQube 10.4
-10.4.sqVersions=[10.4,LATEST]
-10.4.date=2024-02-10
-10.4.changelogUrl=https://github.com/xuhuisheng/sonar-l10n-zh/releases/tag/sonar-l10n-zh-plugin-10.4
-10.4.downloadUrl=https://github.com/xuhuisheng/sonar-l10n-zh/releases/download/sonar-l10n-zh-plugin-10.4/sonar-l10n-zh-plugin-10.4.jar
+10.5.description=Support SonarQube 10.5
+10.5.sqVersions=[10.4,10.LATEST]
```

Comment:
Did you really mean to start this version compatibility at 10.4 and not 10.5?

You can only have one `publicVersion` marked as compatible with any given version of SonarQube.

Reply:
I make a mistake on sqVersions range, I will fix it.